modules/ROOT/pages/import.adoc

```diff
@@ -1464,39 +1464,64 @@ Now use the previously defined ID spaces when connecting the actors to movies.
 == Using multiple node IDs
 
 A node header can contain multiple `ID` columns.
-The relationship data must then use a matching number of `START_ID` / `END_ID` columns as references to the composite value of those ID columns.
+
+Starting from 2025.07, the relationship data must then use a matching number of `START_ID` / `END_ID` columns as references to the composite value of those ID columns.
 This implies using `string` as `id-type`.
 
 For each `ID` column, you can specify to store its values as different node properties.
 However, the composite value cannot be stored as a node property.
 
-[NOTE]
+[IMPORTANT]
 ====
 Incremental import doesn't support the use of multiple node identifiers.
 This functionality is only available with a full import.
 ====
 
 
-.Define multiple IDs as node properties
-====
-You can define multiple `ID` columns in the node header.
-For example, you can define a node header with two `ID` columns.
+=== Define multiple IDs as node properties
 
+. Define multiple `ID` columns in the node header.
++
 .nodes_header.csv
 [source, csv]
 ----
 :ID,:ID,name
 ----
-
++
 .nodes.csv
 [source, csv]
 ----
 aa,11,John
 bb,22,Paul
 ----
 
+. Define the relationship between two established nodes.
++
+[.tabbed-example]
+=====
+[role=include-with-multiple-node-IDs]
```

Comment:
I don't like these headings. In fact, the first tab is on how to use single :START_ID/:END_ID column, and the second one is for matching number of those columns.
Suggestions:
The first tab -- Using single ID columns for composite IDs
The second tab -- Using multiple ID columns for composite IDs